back-end/sports_platform/sports_platform_api/models/event_models.py

```diff
@@ -15,17 +20,47 @@ class Meta:
 
     latitude = models.DecimalField(max_digits=9, decimal_places=6)
     longitude = models.DecimalField(max_digits=9, decimal_places=6)
-    city = models.CharField()
-    district = models.CharField()
+    city = models.CharField(max_length=200)
+    district = models.CharField(max_length=200)
+    country = models.CharField(max_length=200)
 
-    minimumAttendeeCapacity = models.IntegerField(min_value=0)
+    minimumAttendeeCapacity = models.IntegerField()
     maximumAttendeeCapacity = models.IntegerField()
     maxSpectatorCapacity = models.IntegerField()
     minSkillLevel = models.IntegerField()
     maxSkillLevel = models.IntegerField()
 
     created_on = models.DateTimeField()
-    
+
+    def create_event(data):
```

Comment:
I think it would be better to add @staticmethod decorator here

Reply:
Added it, thanks.